src/dropdown/index.ts

```diff
@@ -387,29 +415,39 @@ export class Dropdown extends LitElement {
           color: #333333;
           vertical-align: top;
           width: 180px;
+          min-width: 180px;
+        }
+        .kuc-dropdown__group {
+          border: none;
+          padding: 0px;
+          height: auto;
+          display: inline-block;
+          width: 100%;
+          margin-inline-start: 0px;
+          margin-inline-end: 0px;
```

Comment:
Please change the order of this style description under the "kuc-dropdown[hidden]".

Reply:
OK